### ISSUE TITLE
WMS-Inspector doesn't work on latest verions of Firefox 

### ISSUE THREAD

**Issue by gis-haderslev** [amercader/WMS-Inspector#35]
I've tried on Firefox 44.2 without succes. Absolutely no reaction when I press the icons

**Comment by hellokarl** [comment]
Agreed - FF 47.01 and I cannot get it to work.  Any chance we can get an update?

**Comment by amercader** [comment]
Hi, apologies @gis-haderslev I missed your first message.

Unfortunately I'm no longer working day to day with WMS services and haven't been involved in Firefox extensions building for many years, so I won't be updating this repo any time soon. I'll send a notice asking for new maintainer and mark the extension as not maintained.

Sorry about that.
